src/resources/resources.go

```diff
@@ -0,0 +1,39 @@
+package resources
+
+import (
+	cappv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+	"github.com/dana-team/platform-backend/src/types"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
maybe this package should be under utils

Reply:
No problem